FormlyMapper/FormlyField/FormlyFieldFactory.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+namespace Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+
+use Linio\DynamicFormBundle\FormlyMapper\FormlyField;
+use Linio\DynamicFormBundle\FormlyMapper\FormlyFieldInterface;
+
+class FormlyFieldFactory
+{
+    /**
+     * @var array
+     */
+    protected $fieldConfiguration;
+
+    /**
+     * @var array
+     */
+    protected $formlyFieldConfiguration;
+
+    /**
+     * @var FormlyFieldInterface[]
+     */
+    protected $formlyFields;
+
+    /**
+     * @param int                  $alias
+     * @param FormlyFieldInterface $formlyField
+     */
+    public function addFormlyField($alias, FormlyFieldInterface $formlyField)
+    {
+        $this->formlyFields[$alias] = $formlyField;
+    }
+
+    /**
+     * @param  int $alias
+     *
+     * @return bool
+     */
+    public function has($alias)
+    {
+        return isset($this->formlyFields[$alias]);
+    }
+
+    /**
+     * @param int                   $alias
```

Comment:
`$alias` should not be so far to the right. There is nothing else for it to align to.